src/braket/circuits/gates.py

```diff
@@ -1186,7 +1397,7 @@ def cphaseshift(
         """Registers this function into the circuit class.
 
         Args:
-            control (QubitSetInput): Control qubit(s).
+            control (Optional[QubitSetInput]): Control qubit(s). Default None.
```

Comment:
Ditto; also, shouldn't the last qubit be absorbed into the gate itself?
```suggestion
            control (QubitSetInput): Control qubit(s). The last control qubit
                is absorbed into the target of the instruction.
```